README.md

```diff
@@ -1,4 +1,15 @@
 # icgc-download-client
 Universal download client for ICGC data residing in various environments. 
 
+Using the python script:
```

Comment:
Don't be afraid to use all the formatting features that [Markdown affords](https://guides.github.com/features/mastering-markdown/).